### ISSUE TITLE
Documentation for custom 503 page setup

### ISSUE THREAD

**Issue by derwaldgeist** [zodern/meteor-up#1146]
First of all, I love mup. It just works, which is awesome!

One thing that I am missing is an in-depth documentation about how you would re-configure mup to setup a custom 503 maintenance page. Since I am updating my app quite often, and there is some significant downtime while doing this, a nice "we're updating" page would be great. I saw that there's a way to add some custom nginx config, but I am wondering how I would upload a 503 html page so nginx can actually serve it?

It would be awesome if this was documented in a step-by-step manner, as it's a pretty common requirement, I guess.

**Comment by Batistleman** [comment]
Yes, when doing a redeploy the server requests return a 502 Bad gateway... How can I setup a 503 (even without a custom html page)?

**Comment by Batistleman** [comment]
@derwaldgeist Hi, do you find out how to do this? (I'm not even getting a 503 while updating... I get a 502)

**Comment by jankapunkt** [comment]
@derwaldgeist your post implies to me that there is such functionality already included but just not documented or do I misunderstand and the feature is not there at all right now?
